run/README.namelist

```diff
@@ -1588,6 +1600,7 @@ The following are for observation nudging:
                                      = 2       ; for running the WRF-NMM with its gravity wave drag
                                      = 3       ; NOAA/GSL gravity wave drag and turbulent orographic form drag
  gwd_diags                           = 0       ; set to 1 to output diagnostics for gwd_opt = 3
+ max_rot_angle_gwd                   = 22.5    ! (default) max projection rotation angle permitted for gravity wave drag
```

Comment:
@kkeene44 This doesn't seem to be used any more, probably since 4.0. We should remove it from registry instead.

Reply:
Ok, I removed it, and will remove from the UG too. I'll create a separate PR to remove this from the Registry.